platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -145,6 +142,13 @@ private GetTransactionResponse patchTransaction(PatchTransactionRequest patch)
           sep24Transaction.setMemo(memo);
           sep24Transaction.setMemoType(memoTypeAsString(MEMO_HASH));
         }
+        if (patch.getTransaction().getStellarTransactions() != null) {
+          patch.getTransaction().getStellarTransactions().stream()
+              .max(Comparator.comparingLong(x -> x.getCreatedAt().toEpochMilli()))
+              .ifPresent(
+                  stellarTransaction ->
+                      sep24Transaction.setStellarTransactionId(stellarTransaction.getId()));
+        }
```

Comment:
Do we not want to do this for SEP-31 transaction as well? Isn't this doing what is supposed to be happening [here](https://github.com/stellar/java-stellar-anchor-sdk/blob/d616253c0157b0bc9b67e16cb99811e09a1bd33a/platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java#L200)?

Reply:
We probably do